apricot/protos/apricot.proto

```diff
@@ -131,6 +132,14 @@ message ComponentEntriesResponse {
     repeated string payload = 1;
 }
 
+message HostGetRequest {
+    string prefix = 1;
```

Comment:
I'd propose an alternative structure here. The `keyPrefix` is an important piece of information when you query Consul as your data backend, but we should not expose it to the user. In other words, the user knows whether they want
1) all flps, or
2) only flps assigned to a certain `detector`.

Therefore:
```
message HostGetRequest {
    string detector = 1;
}
```

Now, in this instance we can take advantage of the fact that all Protobuf fields are optional, as well as having well-defined null values. If the `detector` string you get is empty, you return all FLPs. Otherwise, you only return those for a given detector. In both cases, the `keyPrefix` is a piece of hardcoded data-structure information that should stay within Apricot.